modules/update-upgrading-oc-adm-upgrade-status.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-upgrading-oc-adm-upgrade-status_{context}"]
+= Retrieving information about a cluster update using oc adm upgrade status (Technology Preview)
+
+When updating your cluster, it is useful to understand how your update is progressing. While the `oc adm upgrade` command returns limited information about the status of your update, this release introduces the `oc adm upgrade status` command as a Technology Preview feature. This command decouples status information from the `oc adm upgrade` command and provides specific information regarding a cluster update, including the status of the control plane and worker node updates.
+
+The `oc adm upgrade status` command is read-only and will never alter any state in your cluster. 
+
+:FeatureName: The oc adm upgrade status command
+include::snippets/technology-preview.adoc[]
+
+The `oc adm upgrade status` command can be used for clusters from version 4.12 up to the latest supported release. 
+
+[IMPORTANT]
+====
+While your cluster does not need to be a Technology Preview-enabled cluster, you must enable the `oc adm upgrade status` Technology Preview environment variable, otherwise the `oc` will not recognize the command and you will not be able to use the feature. 
+====
+
+.Procedure 
+
+* To enable the `oc adm upgrade status` Technology Preview environment variable, set the `OC_ENABLE_CMD_UPGRADE_STATUS` environmental variable to `true` by running the following command:
++
+[source,terminal]
+----
+$ OC_ENABLE_CMD_UPGRADE_STATUS=true ./oc adm upgrade status
+----
++
+[NOTE]
+====
+This command sets the environment variable to `true` and then performs the `oc adm upgrade status` command. You do not need to include `$ OC_ENABLE_CMD_UPGRADE_STATUS=true` for future uses of `oc adm upgrade status` once the environment variable has been set.
```

Comment:
You don't need the `$` prompt when mentioning the variable.

But also I don't think including it as single pass / the one-line form actually sets it globally. If you want to say this, then I believe you'll need to switch to the two-step procedure, with step 1 being:

```
$ export OC_ENABLE_CMD_UPGRADE_STATUS=true
```
and step 2 being:
```
$ oc adm upgrade status
```